pnpm-lock.yaml

```diff
@@ -904,13 +904,6 @@ packages:
       '@algolia/requester-common': 4.17.0
     dev: false
 
-  /@ampproject/remapping@2.2.0:
```

Comment:
Why has the package lock changed in this PR? No dependency changes. Let's revert this file

Reply:
I rerun install on original lock and the change is gone now